packages/web-components/src/player-component/player-component.style.ts

```diff
@@ -278,10 +278,30 @@ export const styles = css`
         background: black;
     }
 
-    .fullscreen video {
+    :host(.live-off.timeline-on.bottom-controls-on) .fullscreen video {
         max-height: calc(100% - ${bottomControlHeight} - ${timelineHeight});
     }
 
+    :host(.live-off.timeline-on.bottom-controls-off) .fullscreen video {
+        max-height: calc(100% - ${timelineHeight});
+    }
+
+    :host(.live-off.timeline-off.bottom-controls-on) .fullscreen video {
+        max-height: calc(100% - ${bottomControlHeight});
+    }
+
+    :host(.live-off.timeline-off.bottom-controls-off) .fullscreen video {
+        max-height: calc(100%);
+    }
+
+    :host(.live-on.bottom-controls-on) .fullscreen video {
+        max-height: calc(100% - ${bottomControlHeight});
+    }
+
+    :host(.live-on.bottom-controls-off) .fullscreen video {
```

Comment:
May be we should group similar rules together using comma separation rules

Reply:
Sure, I will do that. 
And I will also do that and update the code in PR #100.

Reply:
The similar rules are grouped together.